x/gamm/pool-models/stableswap/amm_test.go

```diff
@@ -100,3 +106,104 @@ func TestCFMMInvariantMultiAssets(t *testing.T) {
 		decApproxEq(t, k2, k3, kErrTolerance)
 	}
 }
+
+func (suite *StableSwapTestSuite) Test_StableSwap_CalculateAmountOutAndIn_InverseRelationship(t *testing.T) {
+	type testcase struct {
+		denomOut         string
+		initialPoolOut   int64
+		initialWeightOut int64
+		initialCalcOut   int64
+
+		denomIn         string
+		initialPoolIn   int64
+		initialWeightIn int64
+	}
+
+	// For every test case in testcases, apply a swap fee in swapFeeCases.
+	testcases := []testcase{
+		{
+			denomOut:         "uosmo",
```

Comment:
Just curious, why so?

Reply:
Just to be aligned with the stabelswap abstraction to avoid confusion for new folks reading the tests. Functionally, there is no difference ofc. Let's not block the PR on this. Up to you if you'd like to change